BOJ/15686_정준우.cpp

```diff
@@ -0,0 +1,47 @@
+#include<iostream>
+#include<vector>
+using namespace std;
+
+int N, M, ans;
+vector<pair<int, int> > chicken_pos, home_pos;
+void check_dist(vector<int> pos){
+    int result = 0;
+
+    for(int i = 0; i < home_pos.size(); i++){
+        int short_path = 1e7;
+        for(int j = 0; j < pos.size(); j++){
+            short_path = min(short_path, 
+                            abs(home_pos[i].first - chicken_pos[pos[j]].first) + abs(home_pos[i].second - chicken_pos[pos[j]].second));
+        }
```

Comment:
같은 깊이의 코드인데 탭이 안맞네요!

```suggestion
        for(int j = 0; j < pos.size(); j++){
            short_path = min(short_path, 
            abs(home_pos[i].first - chicken_pos[pos[j]].first) + abs(home_pos[i].second - chicken_pos[pos[j]].second));
        }
```